.fixtures.yml

```diff
@@ -2,9 +2,9 @@ fixtures:
   repositories:
       stdlib:
         repo: "https://github.com/puppetlabs/puppetlabs-stdlib.git"
-        branch: "4.13.1"
+        branch: "v9.4.0"
       augeas_core:
         repo: "https://github.com/puppetlabs/puppetlabs-augeas_core.git"
-        branch: "1.0.4"
+        branch: "v1.4.1"
   symlinks:
```

Comment:
the `symlink` isn't required anymore, puppetlabs_spec_helper will create it for us.